src/Collection/Collection.router.spec.ts

```diff
@@ -783,7 +783,7 @@ describe('Collection router', () => {
       thirdPartyDbCollection = {
         ...collectionAttributesMock,
         urn_suffix: 'thesuffix',
-        third_party_id: 'some:third-party-id',
+        third_party_id: 'urn:decentraland:some:third-party-id',
```

Comment:
Can we put an existing example here?
```suggestion
        third_party_id: 'urn:decentraland:mumbai:collections-thirdparty:third-party-id',
```

Reply:
Sure